src/sentry/testutils/factories.py

```diff
@@ -946,16 +946,22 @@ def store_event(
         data,
         project_id: int,
         assert_no_errors: bool = True,
-        event_type: EventType = EventType.DEFAULT,
+        event_type: EventType | None = None,
         sent_at: datetime | None = None,
     ) -> Event:
         """
         Like `create_event`, but closer to how events are actually
         ingested. Prefer to use this method over `create_event`
         """
-        if event_type == EventType.ERROR:
+
+        # this creates a basic message event
+        if event_type == EventType.DEFAULT:
             data.update({"stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"])})
 
+        # this creates an error event
+        elif event_type == EventType.ERROR:
+            data.update({"exception": [{"value": "BadError"}]})
```

Comment:
Isn't this going to overwrite these values if they were already passed in the data?

Reply:
Hmm like if you've passed both an `event_type` and `data`? Yeah, let me think about that.

Reply:
@wedamija  Actually I guess my thought here was that if you wanted to provide your own data you can totally do that, you just wouldn't pass `event_type`. But if all you care about is that your type is set correctly, you'd pass it.

Reply:
I see, maybe this should be called `default_event_type` or something? I'd probably recommend at least describing the behaviour in a docstring

Reply:
By `default_event_type` do you mean to simply rename `event_type` or are you suggesting a boolean on whether or not it creates a default type of event?

Reply:
Just renaming it